app/src/main/java/space/taran/arkretouch/data/ImageDefaults.kt

```diff
@@ -0,0 +1,31 @@
+package space.taran.arkretouch.data
+
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.IntSize
+import com.google.gson.Gson
+
+data class ImageDefaults(
+    val colorValue: ULong = Color.White.value,
+    val resolution: Resolution = Resolution(0, 0)
```

Comment:
Does `(0,0)` have special meaning? It can't be used as a resolution for an image, maybe better to use explicit type `Resolution?` if you want to have uninitialized state of the variable.

Reply:
I see we use `IntSize.Zero` here which looks like standard type... @mdrlzy could you check this please? I'm not sure if we need `IntSize.Zero` but that might be standard Android practice..

Reply:
I think it's better to leave `Resolution`, but add `Resolution.Zero` too
So it will be clearer
But then we need to remove the mapping in `IntSize`